tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py

```diff
@@ -76,6 +104,24 @@ def testDatasetCompression(self, element):
     dataset = dataset.map(lambda x: compression_ops.uncompress(x, element_spec))
     self.assertDatasetProduces(dataset, [element])
 
+  def testRaggedCompression(self):
```

Comment:
what happened to the earlier approach of passing _eager_only_test_objects containing ragged tensors as parameters to the existing tests? If that doesn't work for some reason, I would like to understand why before duplicating the test body.

Reply:
@aaudiber , there was an internal build issue with the previous approach where we were passing the `_eager_only_test_objects` to the `@combinations` decorator with the eager test base.

```
TypeError: An op outside of the function building code is being passed
a "Graph" tensor. It is possible to have Graph tensors
leak out of the function building context by including a
tf.init_scope in your function building code.
For example, the following function will fail:
  @tf.function
  def has_init_scope():
    my_constant = tf.constant(1.)
    with tf.init_scope():
      added = my_constant * 2
The graph tensor has name: RaggedConstant/Const:0
```

I was assuming that the test cases coming from the eagerly initialized test base was causing this issue.
And, since the tests pass in both the scenarios (with and without `@combinations`  in a local env) I wanted to cross-check with the approach where we don't use `@combinations`.

Let me know if we can approach this differently.

Reply:
If you write the tests using @combinations, I can run the test internally to understand the problem.

Reply:
addressed in the new commit

Reply:
@aaudiber, did you get a chance to check the internal tests?

Reply:
It seems that the error only happens when using ragged tensors in TF1. Can you exclude that test combination from the test? Currently these compression ops are only used in TF2 code anyway.

Reply:
addressed in the commit